doc/api/v8.md

```diff
@@ -41,19 +41,23 @@ added: v12.8.0
 
 * Returns: {Object}
 
-Returns an object with the following properties:
+Get statistics about code and its metadata in the heap, see V8
+`GetHeapCodeAndMetadataStatistics` API. Returns an object with the
```

Comment:
```suggestion
[`GetHeapCodeAndMetadataStatistics`][] API. Returns an object with the
```
and let's add
```
[`GetHeapCodeAndMetadataStatistics`]: https://v8docs.nodesource.com/node-13.2/d5/dda/classv8_1_1_isolate.html#a6079122af17612ef54ef3348ce170866
```
between these two lines at the bottom of this file https://github.com/nodejs/node/blob/daae938f32f2660a8cb7bf42891bc74b984c7f36/doc/api/v8.md?plain=1#L884-L885

Reply:
done, thanks.